src/index.ts

```diff
@@ -19,6 +19,32 @@ declare module '@typescript-eslint/utils/dist/ts-eslint/Rule' {
   }
 }
 
+declare module '@typescript-eslint/utils/dist/ts-eslint/SourceCode' {
```

Comment:
can we avoid this by upgrading to v6?

Reply:
Yeah I implemented the types in https://github.com/typescript-eslint/typescript-eslint/pull/7812

Reply:
sweet! we should probably do that in a major